src/Interactive/EditorFeatures/CSharp/Completion/FileSystem/ReferenceDirectiveCompletionProvider.cs

```diff
@@ -15,28 +14,9 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.Completion.FileSystem
     // regular C# contexts.  We will need to remove this and implement a new "CSharp Script" Content type
     // in order to fix #r completion in .csx files (https://github.com/dotnet/roslyn/issues/5325).
     [TextViewRole(PredefinedInteractiveTextViewRoles.InteractiveTextViewRole)]
-    internal partial class ReferenceDirectiveCompletionProvider : AbstractReferenceDirectiveCompletionProvider
+    internal sealed class ReferenceDirectiveCompletionProvider : AbstractReferenceDirectiveCompletionProvider
     {
-        protected override bool TryGetStringLiteralToken(SyntaxTree tree, int position, out SyntaxToken stringLiteral, CancellationToken cancellationToken)
-        {
-            if (tree.IsEntirelyWithinStringLiteral(position, cancellationToken))
-            {
-                var token = tree.GetRoot(cancellationToken).FindToken(position, findInsideTrivia: true);
-                if (token.Kind() == SyntaxKind.EndOfDirectiveToken || token.Kind() == SyntaxKind.EndOfFileToken)
-                {
-                    token = token.GetPreviousToken(includeSkipped: true, includeDirectives: true);
-                }
-
-                if (token.Kind() == SyntaxKind.StringLiteralToken &&
-                    token.Parent.Kind() == SyntaxKind.ReferenceDirectiveTrivia)
-                {
-                    stringLiteral = token;
-                    return true;
-                }
-            }
-
-            stringLiteral = default(SyntaxToken);
-            return false;
-        }
+        protected override bool TryGetStringLiteralToken(SyntaxTree tree, int position, out SyntaxToken stringLiteral, CancellationToken cancellationToken) => 
```

Comment:
It would be nice to have this convention captured in .editorconfig.

---
In reply to: [114862678](https://github.com/dotnet/roslyn/pull/19195#discussion_r114862678) [](ancestors = 114862678)